tools/packaging/kata-deploy/local-build/kata-deploy-secure-execution.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash -eu
+# Copyright (c) 2021 IBM Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[ -n "${DEBUG:-}" ] && set -x
+
+kata_build_dir=build
+cmdline="loglevel=0 panic= scsi_mod.scan=none swiotlb=262144 agent.config_file=/etc/kata-config.toml"
+
+usage() {
+	cat >&2 << EOF
+Usage:
+  $0 [options]
+
+Options:
+  -b <build dir>     : Kata build directory, containing kata-static tarballs
+                       Default: "$kata_build_dir"
+  -c <kernel cmdline>: Guest kernel command line
+                       Default: "$cmdline"
+  -h                 : Show this help
+
+Environment variables:
+  HKD (required): Secure Execution host key document, generally specific to your machine. See
+                  https://www.ibm.com/docs/en/linux-on-systems?topic=tasks-verify-host-key-document
+                  for information on how to retrieve and verify this document.
+  DEBUG         : If set, display debug information.
+EOF
+	exit "${1:-0}"
+}
+
+while getopts "b:c:h" opt; do
+	case $opt in
+		b) kata_build_dir="$OPTARG";;
+		c) cmdline="$OPTARG";;
+		h) usage;;
+		*) usage 1;;
+	esac
+done
+shift $(( OPTIND - 1 ))
+
+[ -n "$HKD" ] || (echo >&2 "No host key document specified." && usage 1)
+command -v genprotimg || (echo >&2 "genprotimg is not installed. Install s390-tools." && usage 1)
```

Comment:
Oh - is this PR arch-specific? If so, please add a check on `uname -m` and die if the value is unexpected.

Reply:
Secure Execution is the TEE on s390x, yes. _Right now_, this PR is architecture-specific, but not because you can't get `s390-tools` on other platforms (you can), but because the builds of initrd etc. will be native, so while all architectures could run this, only a result produced by s390x would work. I hope to enable cross builds one day, but that's more to come. 

For now, it is probably better to add a catch, yes. Will update.